docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:11.0.5-jre-slim-buster
```

Comment:
why jre-slim-buster? Besu and EthSigner use the jre-slim-stretch image

Reply:
Aah I understand now these correspond to Debian releases with buster being the latest release. So I think this makes sense to use jre-slim-buster afterall.